src/main/java/frc/robot/constants/AbstractConstants.java

```diff
@@ -184,6 +260,66 @@ public int getGyroId() {
 
     // #endregion
 
+    // #region: --------------- Motor / Motor Controller Settings --------------
+
+    // #region: ----- Falcon 500 Motor -----
+
+    /**
+     * Allow 40A continuous, 80A momentary supply current. See: <a href=
+     * "https://api.ctr-electronics.com/phoenix6/release/java/com/ctre/phoenix6/configs/CurrentLimitsConfigs.html">CTR
+     * Electronics: CurrentLimitsConfigs</a>
+     * 
+     * @return A {@link CurrentLimitsConfigs} object with the default Current
+     *         limits.
+     */
+    public CurrentLimitsConfigs getFalcon500CurrentLimitsConfigs() {
+        CurrentLimitsConfigs limits = new CurrentLimitsConfigs();
+        limits.SupplyCurrentLimitEnable = true;
+        limits.SupplyCurrentLimit = 40.0;
+        limits.SupplyCurrentThreshold = 80.0;
+        limits.SupplyTimeThreshold = 0.5;
+        return limits;
+    }
+
+    // #endregion
+
+    // #region: ----- NEO 550 Brushless Motor -----
+
+    /**
+     * @return Value as Amps.
+     */
+    public int getNeo550BrushlessCurrentLimit() {
+        return 24;
+    }
+
+    /**
+     * @return Value as Amps.
+     */
+    public int getNeo550BrushlessCurrentSecondaryLimit() {
+        return 80;
+    }
+
+    // #endregion
+
+    // #region: ----- Talon FX Motor Controller -----
+    /**
+     * @return Value in Times per Second.
+     */
+    public double getTalonFxStatusSignalUpdateFrequencyDefault() {
+        return 50;
+    }
+
+    /**
+     * This higher rate is needed by PathPlanner.
+     * 
+     * @return Value in Times per Second.
+     */
+    public double getTalonFxStatusSignalUpdateFrequencyForOdometry() {
+        return 100;
```

Comment:
I would like for this to be the same across supported hardware configurations (it's a software choice vs. a property of the hardware).

Reply:
I renamed these to PathPlanner settings.